audio/pcm_decode.c

```diff
@@ -302,6 +303,25 @@ static uint16_t pcm_leuint32(uint32_t value)
 }
 #endif
 
+/****************************************************************************
+ * Name: pcm_leuint32_unaligned
+ *
+ * Description:
+ *   Get a 32-bit value stored in little endian order.  Unaligned address is
+ *   acceptable.
+ *
+ ****************************************************************************/
+
+static uint32_t pcm_leuint32_unaligned(FAR const uint32_t *ptr)
```

Comment:
should we call pcm_leuint32_unaligned(rename to pcm_leuint32) in all place? I am afraid that the alignment happen in other place too. pcm_leuint16 may have the same problem.

Reply:
I see.  I have modified the patch to support unaligned access in both pcm_leuint32() and pcm_leuint16().